pkg/storage/tpr/storage_interface_test.go

```diff
@@ -361,6 +360,37 @@ func TestDeleteNonExistent(t *testing.T) {
 	}
 }
 
+func TestDelete(t *testing.T) {
```

Comment:
Looks good - how about another test (can be in a different PR, I know you are very busy this week) that does a delete on an object that doesn't exist?